crates/hc_sandbox/src/calls.rs

```diff
@@ -418,64 +417,82 @@ async fn call_inner(client: &mut AdminWebsocket, call: AdminRequestCli) -> anyho
                     },
                 }])
                 .await?;
-            msg!("Added admin port {}", port);
+            println!("{}", serde_json::to_value(port)?);
         }
         AdminRequestCli::AddAppWs(args) => {
             let port = args.port.unwrap_or(0);
             let port = client
                 .attach_app_interface(port, args.allowed_origins, args.installed_app_id)
                 .await?;
-            msg!("Added app port {}", port);
+            println!("{}", serde_json::to_value(port)?);
         }
         AdminRequestCli::ListAppWs => {
             let interface_infos = client.list_app_interfaces().await?;
-            msg!("Attached app interfaces {:?}", interface_infos);
+            println!("{}", serde_json::to_value(&interface_infos)?);
         }
         AdminRequestCli::RegisterDna(args) => {
-            let dnas = register_dna(client, args).await?;
-            msg!("Registered DNA: {:?}", dnas);
+            let dna = register_dna(client, args).await?;
+            println!("{}", serde_json::to_value(dna.to_string())?);
         }
         AdminRequestCli::InstallApp(args) => {
             let app = install_app_bundle(client, args).await?;
-            msg!("Installed app: {}", app.installed_app_id,);
+            println!("{}", app_info_to_json(app)?);
         }
         AdminRequestCli::UninstallApp(args) => {
             client
                 .uninstall_app(args.app_id.clone(), args.force)
                 .await?;
-            msg!("Uninstalled app: {}", args.app_id,);
+            msg!("Uninstalled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::ListDnas => {
-            let dnas = client.list_dnas().await?;
-            msg!("DNAs: {:?}", dnas);
+            let dnas: Vec<DnaHashB64> = client
+                .list_dnas()
+                .await?
+                .into_iter()
+                .map(|d| d.into())
+                .collect();
+            println!("{}", serde_json::to_value(&dnas)?);
         }
         AdminRequestCli::NewAgent => {
             let agent = client.generate_agent_pub_key().await?;
-            msg!("Added agent {}", agent);
+            println!("{}", serde_json::to_value(agent.to_string())?);
         }
         AdminRequestCli::ListCells => {
```

Comment:
The display of this isn't clear, I'm seeing

```shell
[
  [
    "uhC0kQdsfrTwLmRR26atXw-cym6Euz0D178ETfKyHjRQ1hT6mNosd",
    "uhCAkErn6N1kBYVHHhbPq_KySEOhhW_s_Ffd91ryN1fxcdYVqfo8E"
  ]
]
```

Previously this would have looked like `DnaHash("uhC0kQdsfrTwLmRR26atXw-cym6Euz0D178ETfKyHjRQ1hT6mNosd")` and similar for the agent key. I know the JS client translates the type this way but if it's going to be transformed for display on the command line then I think making it a structured type with field names would be easier to read.

Reply:
yeah it depends how much the user already knows how a CellId is structured. I can change and have explicit fields instead.

Reply:
I changed the output example